src/script/script_error.h

```diff
@@ -89,6 +89,11 @@ enum class ScriptError {
        marker */
     INVALID_OP_SCRIPTTYPE,
 
+    /* Key spend path for Taproot must use BIP341 sighash */
```

Comment:
I would just expand this comment, and use doc comments here. Note that it's prohibited from EvalScript, but is used to pass through given the existing flag infrastructure for CheckSig* stuff.

Reply:
I think you mean `SCRIPT_TAPROOT_KEY_SPEND_PATH`, not these errors?

Reply:
![image](https://user-images.githubusercontent.com/213740/114257955-3046be80-9978-11eb-9ea6-d5020654172e.png)